website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -449,11 +421,9 @@ make undeploy
 
 ## Further steps
 
-The following guides build off the operator created in this example, adding advanced features:
```

Comment:
I like this bullet formatting better. It could also be numbered, i.e.
```md
Next, try adding the following to your project:
1. Validating and mutating [admission webhooks][create_a_webhook].
2. Operator packaging and distribution with [OLM][olm-integration].

Also see the [advanced topics][advanced_topics] doc for more use cases and a peak under-the-hood of operator-sdk.
```